sys/linux/bpf.txt

```diff
@@ -126,6 +126,8 @@ bpf_prog {
 	line_info_rec_size	const[BPF_LINE_INFO_SIZE, int32]
 	line_info		ptr64[in, bpf_line_info]
 	line_info_cnt		len[line_info, int32]
+	attach_btf_id		bpf_btf_id
+	attach_prog_fd		fd_bpf_prog
```

Comment:
I think it makes sense to mark both of them as `opt`. Otherwise the main way to create fd_bpf_prog requires another fd_bpf_prog, then how does one create them at all? :)

Reply:
Ouch! I updated the branch to fix this.